hazelcast-integration/openshift/README.md

```diff
@@ -288,29 +279,23 @@ Let's first change the directory to Hazelcast Enterprise (`$ cd hazelcast-cluste
 Then, replace ConfigMap with the Persistent Storage.
 
 ```
-oc volume rc/hz-rc --add --overwrite --claim-size 1Gi --mount-path /data/hazelcast
+oc set volume statefulset.apps/hazelcast --add --overwrite --claim-size 1Gi --mount-path /data/hazelcast
 ```
 
-Now, copy `hazelcast.xml` and `ocp-entry-processor-0.1-SNAPSHOT.jar` into the volume. In case of Minishift, you need to first check the volume name.
-
 ```
-$ oc get pvc
-NAME        STATUS    VOLUME    CAPACITY   ACCESS MODES   STORAGECLASS   AGE
-pvc-6qrqc   Bound     pv0090    100Gi      RWO,ROX,RWX                   1h
+cp ../../client-apps/ocp-entry-processor/target/ocp-entry-processor-0.1-SNAPSHOT.jar hazelcast-configuration/
 ```
 
-The volume name is `pv0090` and the following commands copy the necessary files.
+Then, after restarting all PODs (delete them, they will automatically start again), new volume will be available to the pods.
 
 ```
-$ scp -i $HOME/.minishift/machines/minishift/id_rsa hazelcast-configuration/hazelcast.xml docker@$(minishift ip):/mnt/sda1/var/lib/minishift/openshift.local.pv/pv0090/
-$ scp -i $HOME/.minishift/machines/minishift/id_rsa ../../client-apps/ocp-entry-processor/target/ocp-entry-processor-0.1-SNAPSHOT.jar docker@$(minishift ip):/mnt/sda1/var/lib/minishift/openshift.local.pv/pv0090/
+oc delete po --all
 ```
 
-Short explanation of the commands above:
-* `$HOME/.minishift/machines/minishift/id_rsa` - ssh key to Minishift VM is stored in the Minishift's home directory
-* `hazelcast.xml` - custom configuration of Hazelcast
-* `minishift ip` - command to return the IP address of the Minishift VM
-* `/mnt/sda1/var/lib/minishift/openshift.local.pv/pv0090/` - location of the Persistent Volume `pv0090` in Minishift VM
+Now, copy `hazelcast.xml` and `ocp-entry-processor-0.1-SNAPSHOT.jar` into the volume.
```

Comment:
I see `hazelcast.xml` is exactly the same as the one from the `*.yaml` templates. How about removing `hazelcast.xml` at all and leave this section only to the custom entry processors?

Reply:
That was actually needed because when you override `/data/hazelcast` with a new volume (see `oc set volume` command above), there is no hazelcast.xml inside the folder anymore. In fact, old `scp` command was copying a local version to the pod too.